src/components/catsPage/catsPage.scss

```diff
@@ -0,0 +1,51 @@
+:root {
+    font-family: sans-serif;
+  }
+  
+  * {
+    box-sizing: border-box;
+  }
+  
+  .screensaver {
+    font-family: sans-serif;
+    font-size: 1rem;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: hidden;
+  }
+  
+  .screensaver__bouncer {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+  
+  .screensaver__dvd {
+    font-size: calc(1em / 16 * 50);
+    letter-spacing: -0.1em;
+    text-indent: -0.16em;
+    font-weight: bold;
+    text-align: center;
+    margin-bottom: -0.2em;
+  }
+  
+  .screensaver__disc {
+    width: 80%;
+    margin: 0 auto;
+    padding-bottom: 4%;
+    border-style: solid;
+    border-color: black;
+    border-width: 10px 40px;
+    border-radius: 50%;
+    background: transparent;
+  }
+  
+  .screensaver__video {
+    font-size: calc(1em / 16 * 25);
+    font-weight: bold;
+    text-align: center;
+    text-transform: uppercase;
+  }
```

Comment:
Remove unused CSS classes

Reply:
I think these may be used... css is weird and you can call on diff css classes dynamically. Unless if you were able to prove that deleting the css classes didn't affect the animation